opentracing-api/src/main/java/io/opentracing/Tracer.java

```diff
@@ -23,25 +23,37 @@
     /**
      * Return a new SpanBuilder for a Span with the given `operationName`.
      *
+     * <p>If there is an active Span according to the {@link Tracer#scheduler()}'s
+     * {@link Scheduler#activeContext}, buildSpan will automatically build a {@link References#INFERRED_CHILD_OF}
+     * reference to same.
+     *
      * <p>You can override the operationName later via {@link Span#setOperationName(String)}.
      *
      * <p>A contrived example:
      * <pre>{@code
-      Tracer tracer = ...
-
-      Span parentSpan = tracer.buildSpan("DoWork")
-                              .start();
-
-      Span http = tracer.buildSpan("HandleHTTPRequest")
-                        .asChildOf(parentSpan.context())
-                        .withTag("user_agent", req.UserAgent)
-                        .withTag("lucky_number", 42)
-                        .start();
-      }</pre>
+     *   Tracer tracer = ...
+     *
+     *   // Note: if there is an {@link Scheduler#active()} Span, it will be treated as the parent of workSpan.
+     *   Span workSpan = tracer.buildSpan("DoWork")
```

Comment:
I think there's good reason to keep them separate to show how `asChildOf` works, but I did add a comment.